operator/src/main/resources/scripts/introspectDomain.sh

```diff
@@ -119,19 +119,23 @@ if [ ${DOMAIN_SOURCE_TYPE} == "FromModel" ]; then
     trace "Beginning Model In Image"
     command -v gzip
     if [ $? -ne 0 ] ; then
-      trace SEVERE "gzip is missing - image must have gzip installed " && exit 1
+      trace SEVERE "DomainSourceType is FromModel, gzip is missing in the image. Please use an image with gzip installed" && exit 1
     fi
     command -v tar
     if [ $? -ne 0 ] ; then
-      trace SEVERE "tar is missing - image must have tar installed " && exit 1
+      trace SEVERE "DomainSourceType is FromModel, tar is missing in the image. Please use an image with tar installed" && exit 1
+    fi
+    command -v unzip
+    if [ $? -ne 0 ] ; then
+      trace SEVERE "DomainSourceType is FromModel, unzip is missing in the image. Please use an image with unzip installed" && exit 1
     fi
     mkdir -p ${DOMAIN_HOME}
     if [ $? -ne 0 ] ; then
-      trace SEVERE "cannot create domain home directory '${DOMAIN_HOME}'" && exit 1
+      trace SEVERE "DomainSourceType is FromModel, cannot create domain home directory '${DOMAIN_HOME}'" && exit 1
     fi
```

Comment:
Minor nit:  It'd be nice to put a test file in the domain home directory just to verify that it's  accessible - so we can  'fail fast' when hitting a directory access issue.

touch ${DOMAIN_HOME}/testaccess.tmp
if [ $? -ne 0 ]; then
   yadayada Cannot write to domain home directory '${DOMAIN_HOME}' yadayayada
fi
rm -f ${DOMAIN_HOME}/testaccess.tmp

Reply:
done